### PR TITLE
What Data Did Facebook Really Give Cambridge Analytica?

### PR DESCRIPTION
**Pull request by dan-auth0**

### CODE REVIEW

**Comment by scottmathson-auth0** [comment]
screen shot 2018-07-27 at 9 51 58 am
screen shot 2018-07-27 at 9 51 51 am

FYI @dan-auth0 your display photo link is broken.

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Kogan billed “thisisyourdigitallife” as a research app used by psychologists and achieved [270,000 downloads](https://newsroom.fb.com/news/2018/03/suspending-cambridge-analytica/). Each user consented for Kogan to access their personal information, including location, Facebook content they had liked, and friends and connections. 
+
+Cambridge Analytica also obtained data without users' consent from an [additional ~87 million](https://www.wired.com/story/facebook-exposed-87-million-users-to-cambridge-analytica/) accounts and sold their findings to political campaigns, including those of [Ted Cruz](https://www.theguardian.com/us-news/2015/dec/11/senator-ted-cruz-president-campaign-facebook-user-data), Ben Carson, and [Donald Trump](https://www.nytimes.com/2018/03/17/us/politics/cambridge-analytica-trump-campaign.html).
+
+<p style="text-align: center;">
+  <img src="https://cdn.auth0.com/blog/what-data-did-facebook-really-give-cambridge-analytica/Cambridge-Analytica-internal-email.jpg" alt="Cambridge Analytica internal email">
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `Cambridge Analytica internal email`

Suggested: "Cambridge Analytica data breach internal email discussion"

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+</p>
+
+[Source](https://www.nytimes.com/2018/03/17/us/politics/cambridge-analytica-trump-campaign.html)
+
+This data leak is one of the largest and most upsetting for users to date, given the strong evidence of how Cambridge Analytica used their information to influence divisive elections.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `This data leak is one of the largest...`

Suggested: "This data leak and its surrounding scandal is one of the largest..."

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+For the millions of companies that use [Facebook Login](https://auth0.com/learn/social-login/) to authenticate their users, this investigation poses new business and ethical challenges. This article will help teams grapple with how the scandal could affect their operations and offer solutions for moving forward.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `For the millions of companies that use [Facebook Login](https://auth0.com/learn/social-login/) to authenticate...`

Suggested: "For the millions of companies that use [social login](https://auth0.com/learn/social-login/) and in particular [Facebook login](https://auth0.com/docs/connections/social/facebook) to authenticate..."

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+## Should You Stop Using Facebook Login with Your Apps?
+
+Shortly following news of the breach, [Facebook launched an apology ad campaign](https://www.fastcompany.com/40563382/facebook-says-sorry-sort-of-in-its-biggest-ever-ad-campaign) aimed at helping users steer clear of dishonest accounts on its platform that could be attempting to steal their information.
+
+<p style="text-align: center;">
+  <img src="https://cdn.auth0.com/blog/what-data-did-facebook-really-give-cambridge-analytica/apology-ad-campaign.png" alt="Facebook apology ad campaign">
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `Facebook apology ad campaign`

Suggested: "Facebook post-data-breach apology ad campaign"

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+While the campaign provided some good insights, it also highlighted staggering issues within Facebook — particularly how the company has scaled without paying close enough attention to certain user behaviors. 
+
+From fake accounts to clickbait, spam, and more, it appears like [Facebook could be a messy platform to continue to authenticate](https://auth0.com/learn/social-login/) with — at least in the near-term. 
+
+Moreover, it could be a challenge to explain to your users why you are continuing to collect their Facebook data.
+
+### Solution 1: Use Other Forms of Social Login
+
+You have a series of other options for authenticating your users.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `You have a series of other options for authenticating your users.`

Suggested: "Auth0 provides a variety of other options for securely authenticating your users."

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+<p style="text-align: center;">
+  <img src="https://cdn.auth0.com/blog/what-data-did-facebook-really-give-cambridge-analytica/auth0-social-connectors.png" alt="Auth0 Social Connections dashboard">
+</p>
+
+[Auth0 supports 30+ social providers](https://auth0.com/learn/social-login/), including Twitter, Google, Yahoo, and LinkedIn. You can also just add any OAuth2 Authorization Server you need. Despite differences among each provider, Auth0 simplifies the process by unifying the way to call providers and the information you retrieve from them.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `[Auth0 supports 30+ social providers](https://auth0.com/learn/social-login/), including...`

Suggested (alongside previous mention of Auth0): "Some of the 30+ social login providers include: Twitter..."

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+It's simple to switch providers in the [Auth0 dashboard](https://auth0.com/learn/social-login/). Simply click Connections, then Social. To enable a new provider, just flip the switch next to their icon. You can then select the desired attributes and permissions you want to get from the provider in the configuration popup.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Take out irrelevant `[Auth0 dashboard](https://auth0.com/learn/social-login/)` link and replace with something like:

"It's simple to switch providers in the Auth0 dashboard. After signup [signup link opportunity], simply click Connections [dashboard link opportunity], then..."

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+### Solution 2: Incorporate Simpler and More Visible Means of Obtaining User Information
+
+[Progressive profiling](https://auth0.com/blog/progressive-profiling/) is another technique for gradually building up a profile of your customers. Each time they interact with your product, you display a short questionnaire. It can begin with something as simple as requesting a first and last name:
+
+<p style="text-align: center;">
+  <img src="https://cdn.auth0.com/blog/what-data-did-facebook-really-give-cambridge-analytica/auth0-registration-form.png" alt="Auth0 registration page">
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `Auth0 registration page`

Suggested: "Auth0 registration signup with expanded first and last name fields"

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+Starting gradually will help assure your users of your expertise at the outset without overwhelming them with technical information.
+ 
+<p style="text-align: center;">
+  <img src="https://cdn.auth0.com/blog/what-data-did-facebook-really-give-cambridge-analytica/microsoft-form-explaining-usage-of-birthdate.png" alt="Microsoft form explaining usage of birthdate">
+</p>
+
+[Source](https://www.econsultancy.com/blog/69256-gdpr-how-to-create-best-practice-privacy-notices-with-examples)
+
+Adding an automatic text window, like Microsoft does, that appears when requesting data points is a simple yet powerful initial trick. From there, you can prompt viewers to click through to a larger policy statement. If they continue to be curious, place links for them to dig even deeper:
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `...data points is a simple yet powerful initial trick...`

"...data points is a simple and clear enhanced user experience addition."

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+
+<p style="text-align: center;">
+  <img src="https://cdn.auth0.com/blog/what-data-did-facebook-really-give-cambridge-analytica/microsoft-privacy-statement.jpg" alt="Microsoft privacy policy sample">
+</p>
+
+[Source](https://www.econsultancy.com/blog/69256-gdpr-how-to-create-best-practice-privacy-notices-with-examples)
+
+If you're daunted by crafting all of this text yourself — ensuring it is both accurate and digestible —  there are many privacy policy templates online. If you're an e-commerce company, for example, [Shopify](https://www.shopify.com/tools/policy-generator) offers a series of tips and tricks. 
+
+Remember: [investing in current customers](https://blog.ujet.co/30-customer-experience-service-stats-you-cant-ignore) can actually produce better results than attracting new ones. Taking the time to open up to those currently engaged will spur them to create more value for you in the long run.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: I'd remove this external link `[investing in current customers](https://blog.ujet.co/30-customer-experience-service-stats-you-cant-ignore)`

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+
+## Just Because You Rely on Facebook Login, Doesn't Mean There Isn't a Way Forward
+
+Although Facebook has frustrated many companies with its lack of oversight, it still has enormous resources at its fingertips to (hopefully) help it improve. While we wait for the tech titan to rebuild user trust (and perhaps become a beloved tool for business partners once more), the above solutions will help you continue to research, protect, and delight your users.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `...Facebook has frustrated many companies with its lack of oversight...`

Suggested (for more keyword placement throughout): "...Facebook has frustrated many companies in the wake of the Cambridge Analytica data breach scandal, it still..."

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown [hunk 12]
```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "What Data Did Facebook Really Give Cambridge Analytica?"
+description: "Is it time to switch social providers?"
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Suggested: "An honest look at the Cambridge Analytica data breach numbers. What's next for social login authentication? Time to switch social providers?"

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 3 above)
Comment (dan-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: This is in relation to Facebook exclusively though. We should not bundle the other social connections on this statement.

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 5 above)
Comment (dan-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: This one is a blanket statement to indicate Facebook is not the only option. In the next paragraph Auth0 is introduced.

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 6 above)
Comment (dan-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: See above comment, please.

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 9 above)
Comment (dan-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Wording was a bit odd. I updated with this:

> Adding an automatic text window, like Microsoft does, that appears when requesting data points enhances the user experience in a clear and simple way

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 12 above)
Comment (dan-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: @scottmathson-auth0 isn't this title too long?

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 2 above)
Comment (dan-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Reached Scott internally.

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 3 above)
Comment (scottmathson-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Sounds good, thanks for feedback!

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 5 above)
Comment (scottmathson-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Good deal, thanks!

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 12 above)
Comment (scottmathson-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: This is a `description` suggestion and comes in at 140 characters (of the 160 available in SERPs). Good here 👍

File: _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown (hunk 7 above)
Comment (scottmathson-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: @dan-auth0 no inline signup/account create CTA here? What are your thoughts on that? I think it's perfect opportunity to provide signup, before going into tutorial/actual steps. Assume that not everyone reading has an account/provide them a simple path to signup.
https://github.com/auth0/blog/pull/1618/commits/58fd24aa0853fa20bd54fbecf121f75a28dc80f4

Comment (dan-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Not sure how to fit that in there. It's at the end with the Aside.

Comment (scottmathson-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: `asides/about-auth0.markdown` actually doesn't have a signup link there either.
screen shot 2018-07-30 at 1 04 53 pm
Alright, just thought it was a good opportunity for signup link as we have done in other posts. Thanks

Comment (scottmathson-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: Added signup opportunity within the tutorial section of the post: https://github.com/auth0/blog/pull/1618/commits/fa1ee78f69d9b6e22acfa38b27732871abb81039

Comment (scottmathson-auth0) [reply _posts/2018-07-30-what-data-did-facebook-really-give-cambridge-analytica.markdown]: I do see what you're referencing at the end of all posts @dan-auth0 within the `post.html` layout, it includes the `custom-try-banner.html` CTA section. Just thought it'd be worthwhile to include yet another opportunity, inline during this tutorial section. Thanks 😄

**Pull request merged**